dist.ini

```diff
@@ -9,3 +9,14 @@ no_makemaker = 1
 alien_repo = http://www.mpg123.de/download
 alien_pattern_suffix = \.tar\.bz2
 alien_bins = mpg123
+
+[Prereqs / ConfigureRequires]
+Alien::Base = 0.021
+Alien::Base::ModuleBuild = 0.023
+Test::LoadAllModules = 0
+
+[Prereqs / Requires]
+Alien::Base = 0.021
+Alien::Base::ModuleBuild = 0.023
```

Comment:
ABMB isn't a runtime prereq, so it should only be in configure requires.

Reply:
Fixed with new commit, thank you ! :smile: